### PR TITLE
docs: Some typos in database/configuration.rst

### PR DESCRIPTION
**Pull request by obozdag**
**Description**
Some typos in database/configuration.rst

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/database/configuration.rst [hunk 1]
```diff
@@ -29,13 +29,13 @@ prototype:
 The name of the class property is the connection name, and can be used
 while connecting to specify a group name.
 
-.. note:: The default location of the SQLite3 database is in the **writable** folder.
-    If you want to change the location, you must set the full path to the new folder.
+.. note:: The default location of the SQLite3 database is the **writable** folder.
+    If you want to change the location, you must set the full path to the new folder (e.g., 'database' => '../db/database_name').
```
Comment (kenjis) [inline user_guide_src/source/database/configuration.rst:33]: 1. `../db/database_name` is not a full path, but a relative path.
2. It is better to have the file extention.

Change to something like `WRITEPATH . 'db/database_name.db'`.

Comment (obozdag) [reply user_guide_src/source/database/configuration.rst]: @kenjis Changed. Thanks.

**Review by kenjis — APPROVED** [review]
LGTM!

**Review by datamweb — APPROVED** [review]

File: user_guide_src/source/database/query_builder.rst [hunk 2]
```diff
@@ -88,7 +88,7 @@ $query, which can be used to show the results:
 
 .. literalinclude:: query_builder/004.php
 
-Please visit the :doc:`getResult*() method <results>` page for a full
+Please visit the :doc:`getResult() method <results>` page for a full
```
Comment (kenjis) [inline user_guide_src/source/database/query_builder.rst:91]: I agree to remove `*`. The `*` implies `getResultArray()` and `getCustomResultObject()`, but they are referred in the [getResult()](https://codeigniter4.github.io/CodeIgniter4/database/results.html#getresult) section.

But it is better that the destination is [getResult()](https://codeigniter4.github.io/CodeIgniter4/database/results.html#getresult), not the page.
Can you change the destination?
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/documentation.rst#to-a-section

File: user_guide_src/source/database/query_builder.rst [hunk 3]
```diff
@@ -101,14 +101,14 @@ Example:
 
 .. literalinclude:: query_builder/005.php
 
-The first parameter enables you to set whether or not the query builder query
+The parameter in the first query (false) enables you to set whether or not the query builder
```
Comment (kenjis) [inline user_guide_src/source/database/query_builder.rst:104]: I don't get what "the first query" is.

It means "the first parameter of the getCompiledSelect() method".
So "The first parameter" is correct and no problem.

Comment (kenjis) [reply user_guide_src/source/database/query_builder.rst:104]: How about adding the default value for the parameter is `true`?

File: user_guide_src/source/database/query_builder.rst [hunk 4]
```diff
@@ -88,7 +88,7 @@ $query, which can be used to show the results:
 
 .. literalinclude:: query_builder/004.php
 
-Please visit the :doc:`getResult*() method <results>` page for a full
+Please visit the :doc:`getResult() method <getresult>` method for a full
```
Comment (kenjis) [inline user_guide_src/source/database/query_builder.rst:91]:
```suggestion
Please visit the :ref:`getResult() <getresult>` method for a full
```

**Review by kenjis — APPROVED** [review]

**Review by michalsn — APPROVED** [review]

**Pull request merged**